src/Util/Reflection.php

```diff
@@ -37,6 +37,8 @@ public function getProperty($methodName)
         if (preg_match('/^('.$pattern.')(.+)$/i', $methodName, $matches)) {
             return $matches[2];
         }
+        
+        return null;
```

Comment:
Do we have to be this strict on following Symfony CS? I get it's convenient, but in some cases like this one, it just makes the code less readable IMO